frontend/institution/managementMembersController.js

```diff
@@ -250,23 +250,11 @@
             manageMemberCtrl[flagName] = !manageMemberCtrl[flagName];
         };
 
-        manageMemberCtrl.isUserInvitesValid = function isUserInvitesValid(invite, emails) {
+        manageMemberCtrl.isUserInvitesValid = function isUserInvitesValid(invite) {
```

Comment:
change to isUserInviteValid because this function checks only one invite. If using only the function invite.isValid is not enough?